src/test/cypress/support/pageobjects/AssessmentDashboard.ts

```diff
@@ -0,0 +1,31 @@
+export class AssessmentDashboard {
+    openExerciseDashboard() {
+        cy.get('[jhitranslate="entity.action.exerciseDashboard"]').should('be.visible').click();
```

Comment:
Click fails if the element is not visible, so the assertion is redundant
```suggestion
        cy.get('[jhitranslate="entity.action.exerciseDashboard"]').click();
```